src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -58,14 +58,24 @@ public static List<String> getMeasureTypes(Measure measure) {
   }
 
   public static String getStratification(Measure measure) {
+    // Collects and returns all stratification descriptions for display
     if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+      String allDescriptions = "";
       for (Group group : measure.getGroups()) {
+        if (allDescriptions.length() > 0
```

Comment:
```suggestion
        if (!allDescriptions.isEmpty()
```
Since allDescriptions is a string, we could use isEmpty. It does the same thing but helps developers to read better.